tests/post/test_kubectl.py

```diff
@@ -12,6 +12,12 @@ def get_node_list(request, inventory_obj):
     if request.kubectl_result.returncode != 0:
         pytest.fail('Error running kubectl command')
     lines = request.kubectl_result.stdout.read().splitlines()
+    for n, line in enumerate(lines, start=1):
```

Comment:
I don't understand why these changes are required. Is the `kubectl` output ever 'different'?

Also, instead of this, how about changing the test to run `kubectl get nodes -o json`, and `json.loads` the output, then work on the resulting object?

Reply:
Yes because we can have the virtualenv creation (and downloading of kubectl and helm) that does print to stdout